src/racer/scopes.rs

```diff
@@ -366,17 +366,27 @@ fn expand_search_expr_handles_double_dot() {
     assert_eq!((2, 5), expand_search_expr("..foo", 4))
 }
 
+fn fill_gaps(buffer: &str, result: &mut String, start: usize, prev: usize) {
```

Comment:
Is this change integral to the match/if-let work?

Reply:
Yep, I changed the scope bounds for the IfLetVisitor and without these changes one test failed, because one point was out of bounds.